setup/README.md

```diff
@@ -125,7 +125,7 @@ openssl ec -in root-priv.key -pubout -out root.pub
 - After entering your password, copy the public key to the `helm/values.yaml`: 
 
 ```bash
-yq write --inplace ${OLDPWD}/helm/values.yaml -- notary.rootPubKey "$(cat root.pub)"
+yq e '.notary.rootPubKey=$(cat root.pub)' -i ${OLDPWD}/helm/values.yaml
```

Comment:
Was also present before, but if OLD_PW contains whitespace, does this break?

Reply:
just tried it. You're right, it fails! Can we address this case by code or should we add a hint in the paragraph above this snippet?

Reply:
You can emded the path string in quotes